diesel/Cargo.toml

```diff
@@ -15,7 +15,7 @@ categories = ["database"]
 byteorder = "1.0"
 diesel_derives = "~1.3.0"
 chrono = { version = "0.4", optional = true }
-clippy = { optional = true, version = "=0.0.195" }
+clippy = { optional = true, version = "=0.0.211" }
```

Comment:
Please avoid changes that are unrelated to your main PR. There's no reason to update clippy here.